Sources/NIO/System.swift

```diff
@@ -364,10 +364,13 @@ internal enum Posix {
     }
 
     @inline(never)
-    public static func sendto(descriptor: CInt, pointer: UnsafeRawPointer, size: size_t,
+    public static func sendto(descriptor: CInt,
+                              pointer: UnsafeRawPointer,
+                              size: size_t,
+                              flags: CInt,
```

Comment:
~Isn’t this an API breaking change? Or am I missing something?~

How come we have public methods on an internal enum?